docs/source/onnxruntime/usage_guides/models.mdx

```diff
@@ -91,3 +91,24 @@ Here is an example of how you can load a T5 model to the ONNX format and run inf
 >>> result = onnx_translation(text)
 >>> # [{'translation_text': "Il n'est jamais sorti sans un livre sous son bras, et il est souvent revenu avec deux."}]
 ```
+
+## Export and inference of stable diffusion models
+
+Stable diffusion models can also be used when running inference with ONNX Runtime. When Stable diffusion models
+are exported to the ONNX format, they are decomposed into three components that are later combined during inference:
+- The text encoder component
+- The U-NET component
+- The VAE decoder component
+
+Here is an example of how you can load a stable diffusion model, export it to the ONNX format and run inference using ONNX Runtime:
+
+```python
+from optimum.onnxruntime import ORTStableDiffusionPipeline
+
+model_id = "runwayml/stable-diffusion-v1-5"
+stable_diffusion = ORTStableDiffusionPipeline.from_pretrained(model_id, export=True)
```

Comment:
It's a bit tricky to advertise `export=True` given that the cache has been disabled for now and the export for somewhat large models like stable diffusion can be quite slow. So I'd advocate either to add a section about offline loading, or add back a cache in a following PR.